objects/screen.c

```diff
@@ -241,6 +243,143 @@ screens_exist(void)
     return globalconf.screens.len > 0;
 }
 
+#ifdef XCB_RANDR_GET_MONITORS
```

Comment:
Could you replace this with something like the following (is there a nicer way to express this?)
```
/* Monitors were introduced in RandR 1.5 */
#if XCB_RANDR_MAJOR_VERSION > 1 || XCB_RANDR_MINOR_VERSION >= 5
```